lib/iris/_lazy_data.py

```diff
@@ -42,17 +42,21 @@ def _iris_dask_defaults():
     all available CPUs.
 
     .. note::
+
         We only want Iris to set dask options in the case where doing so will
         not change user-specified options that have already been set.
 
     """
+    dask_opts = {}
     if 'pool' not in dask.context._globals and \
```

Comment:
I'd probably guard about accessing ```dask.context._globals```. They could legitimately remove this and leave iris unable to load data... 🤒

Reply:
(to be clear, in this context I simply mean ```getattr(dask.context, '_globals')```)

Reply:
Given the nature of this API (private), I'd also be tempted to add a test that asserts dask's set_opt behaviour continues to be a valid assumption in ```dask.context._globals```.